pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -168,6 +170,11 @@ func (admitter *VMsAdmitter) Admit(ar *admissionv1.AdmissionReview) *admissionv1
 	}
 
 	if ar.Request.Operation == admissionv1.Create {
+		clusterCfg := admitter.ClusterConfig.GetConfig()
```

Comment:
Shouldn't it validate FG first before getting into the spec? (similar to VMI admitter)